MapboxSearch/ui/src/main/java/com/mapbox/search/ui/utils/extenstion/View.kt

```diff
@@ -12,7 +12,11 @@ internal fun View.showKeyboard() {
     }
 }
 
-internal fun View.hideKeyboard() {
+/**
```

Comment:
Why do you need this change? When you develop SDK it's better to keep everything non-public, until proven otherwise.

Reply:
I made it internal again. It was an attempt to avoid some code duplication.